vocabularies/sample-preparation.ttl

```diff
@@ -0,0 +1,269 @@
+    skos:topConceptOf smprep:conceptScheme .
+
+smprep:ionise a skos:Concept ;
+    skos:altLabel "Ionize"@en ;
+    skos:definition "To change a gas to a plasma. Typically through decreasing temperature."@en ;
+    skos:inScheme smprep:conceptScheme ;
+    skos:prefLabel "Ionise"@en ;
+    skos:topConceptOf smprep:conceptScheme .
+
+smprep:laser-ablate a skos:Concept ;
+    skos:definition "Laser ablation is the process of removing material from a solid (or occasionally liquid) surface by irradiating it with a laser beam."@en ;
+    skos:inScheme smprep:conceptScheme ;
+    skos:prefLabel "Laser Ablate"@en ;
+    skos:topConceptOf smprep:conceptScheme .
+
+smprep:melt a skos:Concept ;
+    skos:definition "To change a solid to a liquid. Typically through increasing temperature or decreasing pressure."@en ;
+    skos:inScheme smprep:conceptScheme ;
+    skos:prefLabel "Melt"@en ;
+    skos:topConceptOf smprep:conceptScheme .
+
+smprep:mount a skos:Concept ;
+    skos:definition "To be mounted on or between media in preparation for petroscopy or other observation technique which requires a stable, protected sample."@en ;
+    skos:inScheme smprep:conceptScheme ;
+    skos:prefLabel "Mount"@en ;
+    skos:topConceptOf smprep:conceptScheme .
+
+smprep:precipitate a skos:Concept ;
+    skos:definition "The process of precipitating a material out of a solvent."@en ;
+    skos:inScheme smprep:conceptScheme ;
+    skos:prefLabel "Precipitate"@en ;
+    skos:topConceptOf smprep:conceptScheme .
+
+smprep:react a skos:Concept ;
+    skos:definition "To change the chemical state of a sample by combination with a reagent."@en ;
+    skos:inScheme smprep:conceptScheme ;
+    skos:prefLabel "React"@en ;
+    skos:topConceptOf smprep:conceptScheme .
+
+smprep:recombine a skos:Concept ;
+    skos:definition "To change a gas to a plasma. Typically through increasing temperature."@en ;
+    skos:inScheme smprep:conceptScheme ;
+    skos:prefLabel "Recombine"@en ;
+    skos:topConceptOf smprep:conceptScheme .
+
+smprep:sieve a skos:Concept ;
+    skos:definition "Material characterised by size fractionation through a screen or series of screens."@en ;
+    skos:inScheme smprep:conceptScheme ;
+    skos:prefLabel "Sieve"@en ;
+    skos:topConceptOf smprep:conceptScheme .
+
+smprep:sublimate a skos:Concept ;
+    skos:definition "To change a solid to a gas. Typically through increasing temperature or decreasing pressure."@en ;
+    skos:inScheme smprep:conceptScheme ;
+    skos:prefLabel "Sublimate"@en ;
+    skos:topConceptOf smprep:conceptScheme .
+
+smprep:thermal-treatment a skos:Concept ;
+    skos:altLabel "Bake"@en,
+        "Baking"@en,
+        "Heat"@en,
+        "Heating"@en ;
+    skos:definition "The process of applying thermal radiation to a material in order to chemically alter or oxidise it."@en ;
+    skos:inScheme smprep:conceptScheme ;
+    skos:prefLabel "Thermal treatment"@en ;
+    skos:topConceptOf smprep:conceptScheme .

```

Comment:
Heating and Thermal Treatment are different. 
Need to remove Heat/Heating from Thermal Treatment and implement a new term.

```
smprep:Heating a skos:Concept ;
    skos:altLabel "Heat"@en;
    skos:definition "The process of applying thermal radiation to a material in order to raise its surface and or internal temperature."@en ;
    skos:inScheme smprep:conceptScheme ;
    skos:prefLabel "Heating"@en ;
    skos:topConceptOf smprep:conceptScheme .
```
Need to add it to the list of top terms too.

Reply:
Heat added as concept.
Labels for thermal treatment changed as per above.